### PR TITLE
This should get a basic dev environment up.

### PR DESCRIPTION
**Pull request by jreuter**
It should have a 'Hello World' message, but it doesn't appear to work for some reason.  Their are no errors in the google_appengine output and I don't see any logging happening by default.

### CODE REVIEW

**Comment by animan1** [comment]
LGTM. I don't know enough about Flask to comment on the hello world issue.

**Comment by jreuter** [comment]
@animan1 I got it figured out (or rather Jordon did).  I needed to specify `--host 0.0.0.0` when I run GAE so I can hit it from outside my VM.  I can't believe I didn't see that before.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,20 @@
-kegstarter
+Kegstarter
 ==========
 
 Kegstarter
+
+Developer Setup
+===============
+The developer setup is assuming Ubuntu Server 14.04 as a starting point.
```
Comment (iparanza) [inline README.md:8]: Are we going to set Ubuntu as the development environment?

Comment (zackdouglas) [inline README.md:8]: > Ubuntu as the development environment

I hope not.

Comment (zackdouglas) [inline README.md:8]: This is moved to #3.

**Comment by iparanza** [comment]
Need to get started on this... I'm fine with ubuntu. LGTM

**Comment by jreuter** [comment]
@animan1 @iparanza Whack the button?

**Pull request merged**